src/backend/distributed/commands/create_citus_local_table.c

```diff
@@ -71,17 +71,18 @@ static void InsertMetadataForCitusLocalTable(Oid citusLocalTableId, uint64 shard
 static void FinalizeCitusLocalTableCreation(Oid relationId);
 
 
+PG_FUNCTION_INFO_V1(citus_add_local_table_to_metadata);
```

Comment:
I have a slight preference to rename the file to `citus_add_local_table_to_metadata`.